### PR TITLE
Shiviagarw psedu lidar

### PR DESCRIPTION
**Pull request by Shiviagarw**
Adding summary

### CODE REVIEW

**Comment by zhaoyue-zephyrus** [comment]
@dalmeraz any comments?

File: _summaries/wang2018pseudo-lidar_1.md [hunk 1]
```diff
@@ -1,19 +1,72 @@
...
+				
+								(depth) z = D(u, v) 
+								(width) x = (u − cU ) × z /fU 
+								(height) y = (v − cV ) × z /fV ,
+								
+where (cU , cV ) is the pixel location corresponding to the camera center and fV is the vertical focal length.
+By back-projecting all the pixels into 3D coordinates, a 3D point cloud {(x (n) , y(n) , z(n) )} N n=1, where N is the pixel count is achieved. 
+This is called pseudo-LiDAR signal.
+
+3D object detection: The LiDAR-based 3D object detectors algorithm is applied to the estimated pseudo-LiDAR points. The two algorithms used are AVOD and
```
Comment (dalmeraz) [inline _summaries/wang2018pseudo-lidar_1.md:49]: The sentence "In Frustum pointnet the e pseudo-LiDAR informa tion as a 3D point cloud" has a couple of typos

File: _summaries/wang2018pseudo-lidar_1.md [hunk 2]
```diff
@@ -1,19 +1,72 @@
...
+This is called pseudo-LiDAR signal.
+
+3D object detection: The LiDAR-based 3D object detectors algorithm is applied to the estimated pseudo-LiDAR points. The two algorithms used are AVOD and 
+Frustum PointNet. In Frustum pointnet the e pseudo-LiDAR informa tion as a 3D point cloud. In AVOD the pseudo-LiDAR information is viewed from a Bird’s Eye 
+View (BEV). In particular, the 3D information is converted into a 2D image from the top-down view.
+
+				
+![LeNet](wang2018pseudo-lidar_1a.png)
+
+## Experiments:
```
Comment (dalmeraz) [inline _summaries/wang2018pseudo-lidar_1.md:56]: I would consider adding table 1 as well for a full comparison between all types of model and just mentioning gray results = LiDAR, and blue = pseudo-LiDAR

File: _summaries/wang2018pseudo-lidar_1.md [hunk 3]
```diff
@@ -1,19 +1,72 @@
...
+## Approach and Technical representation:
+
+Despite the many advantages of image-based 3D object recognition, there remains a glaring gap between the stateof-the-art detection rates of image and 
+LiDAR-based approaches. Approach to “close the gap” is therefore to carefully remove the differences between the two data modalities and align the two 
+recognition pipelines as much as possible. To achieve this there is a two step approach where first step is to estimate the depth of the pixels from stereo  
+imagery and the second step is  back-projecting pixels into a 3D point cloud called pseudo lidar signals.
+
+Depth estimation: The depth estimation is done with A stereo disparity estimation algorithm. The depth map is derived by the following transformation.
+
+									D(u, v) = fU × b /Y (u, v)
```
Comment (dalmeraz) [inline _summaries/wang2018pseudo-lidar_1.md:30]: "stateof-the-art" to "state-of-the-art"

File: _summaries/wang2018pseudo-lidar_1.md [hunk 4]
```diff
@@ -1,19 +1,72 @@
...
+
+
+## Approach and Technical representation:
+
+Despite the many advantages of image-based 3D object recognition, there remains a glaring gap between the stateof-the-art detection rates of image and 
+LiDAR-based approaches. Approach to “close the gap” is therefore to carefully remove the differences between the two data modalities and align the two 
+recognition pipelines as much as possible. To achieve this there is a two step approach where first step is to estimate the depth of the pixels from stereo  
+imagery and the second step is  back-projecting pixels into a 3D point cloud called pseudo lidar signals.
+
+Depth estimation: The depth estimation is done with A stereo disparity estimation algorithm. The depth map is derived by the following transformation.
```
Comment (dalmeraz) [inline _summaries/wang2018pseudo-lidar_1.md:28]: "A" to "a"

File: _summaries/wang2018pseudo-lidar_1.md [hunk 5]
```diff
@@ -1,19 +1,72 @@
...
+									D(u, v) = fU × b /Y (u, v)
+
+Where D is the depth map ,Fu is the horizontal focal length,
+B is the horizontal offset baseline and Y is the output disparity map.
+
+It takes a pair of left-right images Il and Ir as input, captured from a pair of cameras with a horizontal offset (i.e., baseline) b, and outputs a disparity 
+map Y of the same size as either one of the two input images.The depth estimation algorithm treats the left image, Il , as reference and records in Y the 
+horizontal disparity to Ir for each pixel.
+
+Pseudo-LiDAR generation: In this spet the 3D location (x,y,z) of each pixel (u,v), is computed by the following formulas:
```
Comment (dalmeraz) [inline _summaries/wang2018pseudo-lidar_1.md:39]: typo with "spet"

**Review by dalmeraz — COMMENTED** [review]
Really good summary! Mainly small typo comments :)

**Comment by dalmeraz** [comment]
> @dalmeraz any comments?

Done

File: _summaries/wang2018pseudo-lidar_1.md [hunk 6]
```diff
@@ -1,19 +1,72 @@
...
+
+## The Core Idea of this paper?
+3D object detection is an important task in autonomous driving.  There are several approaches for 3D object detection based on cheaper monocular or stereo 
+imagery data, but these have resulted in lower accuracies due to poor image based depth estimation. This paper discusses that it is not the quality of data 
+but its representation that causes poor accuracy. The core idea of this paper is to do data representation by  converting image-based depth maps to 
+pseudo-LiDAR representations - essentially mimicking the LiDAR signal. LiDAR is (Light Detection And Ranging), which provide accurate 3D point clouds of 
+the surrounding environment .With such representation there are different LiDAR-based detection algorithms that can be applied.Hence this paper shows that 
+a major cause for the performance gap between stereo-based and LiDAR-based 3D object detection is not the quality of the estimated depth but its 
+representation. Second, the  pseudo-LiDAR as a new recommended representation of estimated depth for 3D object detection and show that it leads to 
+state-of-the-art stereobased 3D object detection, effectively tripling prior art.
```
[the remark this answers is not in the document]
Comment (Shiviagarw) [reply _summaries/wang2018pseudo-lidar_1.md:18]: Done the change.

File: _summaries/wang2018pseudo-lidar_1.md (hunk 4 above)
Comment (Shiviagarw) [reply _summaries/wang2018pseudo-lidar_1.md]: Done the change.

File: _summaries/wang2018pseudo-lidar_1.md (hunk 3 above)
Comment (Shiviagarw) [reply _summaries/wang2018pseudo-lidar_1.md:30]: Done the change.

File: _summaries/wang2018pseudo-lidar_1.md (hunk 5 above)
Comment (Shiviagarw) [reply _summaries/wang2018pseudo-lidar_1.md]: Done the change.

File: _summaries/wang2018pseudo-lidar_1.md (hunk 1 above)
Comment (Shiviagarw) [reply _summaries/wang2018pseudo-lidar_1.md:49]: Done the change.

File: _summaries/wang2018pseudo-lidar_1.md (hunk 2 above)
Comment (Shiviagarw) [reply _summaries/wang2018pseudo-lidar_1.md:56]: Done the change.

**Comment by Shiviagarw** [comment]
Thank you for your review. I have done the changes as suggested by you and
repressed the updated version. Please check.
Shivi Agarwal
(She/her/hers)
Master of Science Computer Science
graduation year(2021-2022) University Of Texas, Austin
***@***.***
Cell # 5125865037

**Review by zhaoyue-zephyrus — APPROVED** [review]

**Pull request merged**